# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

C42_CpgWlkActionGoal action_goal
C42_CpgWlkActionResult action_result
C42_CpgWlkActionFeedback action_feedback
